pkg/trie/triedb/proof/proof_test.go

```diff
@@ -1,22 +1,19 @@
-// Copyright 2024 ChainSafe Systems (ON)
-// SPDX-License-Identifier: LGPL-3.0-only
-
 package proof
```

Comment:
super nit: maybe create a proof.go that hold `MerkleProof` and constructor?

Reply:
I like it! done